array/sort.mbt

```diff
@@ -96,16 +96,15 @@ fn merge[T : Compare](arr : ArraySlice[T], mid : Int) -> Unit {
     buf[i] = arr[mid + i]
   }
   let buf = { array: buf, start: 0, end: buf_len }
-  let buf_remaining = 
-    for p1 = mid - 1, p2 = buf_len - 1, p = mid + buf_len - 1
-        p1 >= 0 && p2 >= 0; {
-      if arr[p1] > buf[p2] {
-        arr[p] = arr[p1]
-        continue p1 - 1, p2, p - 1
-      } else {
-        arr[p] = buf[p2]
-        continue p1, p2 - 1, p - 1
-      }
+  let buf_remaining = for p1 = mid - 1, p2 = buf_len - 1, p = mid + buf_len - 1; p1 >=
```

Comment:
This formatting looks kinda weird

Reply:
Yeah, it's a known issue. We are working on it